nncf/config_schema.py

```diff
@@ -121,9 +121,40 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
     "additionalProperties": False
 }
 
-INITIALIZER_SCHEMA = {
+GENERIC_INITIALIZER_SCHEMA = {
     "type": "object",
     "properties": {
+        "batchnorm_adaptation":
+            {
+                "type": "object",
+                "properties": {
+                    "num_bn_adaptation_steps": with_attributes(_NUMBER,
+                                                               description="Number of batches from the training "
+                                                                           "dataset to use for model inference during "
+                                                                           "the BatchNorm statistics adapation "
+                                                                           "procedure for the compressed model"),
+                },
+                "additionalProperties": False,
+            },
+    },
+    "additionalProperties": False,
+}
+
+QUANTIZATION_INITIALIZER_SCHEMA = {
+    "type": "object",
+    "properties": {
+        "batchnorm_adaptation":
+            {
+                "type": "object",
+                "properties": {
+                    "num_bn_adaptation_steps": with_attributes(_NUMBER,
```

Comment:
it makes sense to mention `bn_forget_steps`

Reply:
Done.